### PR TITLE
import persian number code to number formatted

### PR DESCRIPTION
**Pull request by joshrayman**
Screen Shot 2019-07-23 at 11 24 37
(percentages)

Outstanding:
negative numbers
Screen Shot 2019-07-23 at 11 25 55

### CODE REVIEW

**Review by rushlet — APPROVED** [review]
looks a bit weird, but if it has to be like that to work then 🤷‍♀

File: numberFormatter.js [hunk 1]
```diff
@@ -32,6 +32,35 @@ define(function () {
...
+                    outputStr += persianArray[10];
+                }
+                else {
+                    // try and catch other non-number characters.
+                    outputStr += numStr[i];
+                }
+            }
+
+            return outputStr;
+        }
```
Comment (rushlet) [inline numberFormatter.js]: missed a comma here!

**Review by rushlet — CHANGES_REQUESTED** [review]

File: numberFormatter.js [hunk 2]
```diff
@@ -32,6 +32,35 @@ define(function () {
         'arabic': function (number) {
            return formatNumberWithSeperators(number, '', ',');
         },
+        'persian': function (number) {
+            const persianArray = ['۰', '۱', '۲', '۳', '۴', '۵', '۶', '۷', '۸', '۹', '،'];
+            let outputStr = '';
+
+            const numStr = num.toString();
```
Comment (rushlet) [inline numberFormatter.js]: is `num` supposed to be `number` ?

**Review by rushlet — CHANGES_REQUESTED** [review]

File: numberFormatter.js (hunk 2 above)
Comment (joshrayman) [reply numberFormatter.js]: yes sorry

**Pull request merged**